### PR TITLE
M890 Tactical Shotgun

### PR DESCRIPTION
**Pull request by Cyberanarchbee**
## About the PR

Added the M890 Tactical shotgun to the roster of weapons. It has not been mapped or added to appropriate vendors, merely the item now exists.

## Why / Balance

Mostly for parities sake, CM13 has one, but it would help fill the armouries aboard the Almayer.

## Technical details

Mostly just added the RSI pulled from CM13 and changed the pump shotguns code to be more appropriate to the M890 . Otherwise, should be fairly simple. 

## Media

Screenshot 2024-09-29 155321

The M890 with various attachments and its brand new stock!

## Requirements

- [TICK] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [TICK] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

Added the M890 tactical shotgun and its appropriate stock!

🆑 
- add: M890 Tactical Shotgun
- add: M890 Tactical Shotgun Stock

### CODE REVIEW

**Comment by Cyberanarchbee** [comment]
changes i need to make here: turns out the mk221 is not a useable name, would take suggestions to fit the rest of the themeing,

someone said the stock sits a little low?

**Comment by KittenColony** [comment]
M58A3 for a name

stock shouldn't be an issue but maybe move it up slightly?

**Comment by DrSmugleaf** [comment]
> changes i need to make here: turns out the mk221 is not a useable name, would take suggestions to fit the rest of the themeing,
> 
> someone said the stock sits a little low?

We have it noted down as M890, so replace any usages of MK221 with M890

**Review by Cyberanarchbee — COMMENTED** [review]
M890!!!!

**Comment by Cyberanarchbee** [comment]
shit i changed the name in the code, but github is refusing to allow me to change the rsi name. working on it.

**Comment by Cyberanarchbee** [comment]
RSI check supposedly failed, same with YAML, looks fine on my end.

File: Resources/Textures/_RMC14/Objects/Weapons/Guns/Shotguns/m890.rsi/meta.json [hunk 1]
```diff
@@ -1,32 +1,40 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/4047c473d4d2f3e58e68efdec5aa95d0676bf4d1/icons/obj/items/weapons/guns/guns_by_faction/uscm.dmi, https://github.com/cmss13-devs/cmss13/blob/106c92cdf232ebc12c9d7a2feb23956c6755496f/icons/mob/humans/onmob/items_lefthand_1.dmi, https://github.com/cmss13-devs/cmss13/blob/4047c473d4d2f3e58e68efdec5aa95d0676bf4d1/icons/mob/humans/onmob/items_righthand_1.dmi",
+  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/7cb618c69b75873f3ce893022fe08d1233b3152d/icons/obj/items/weapons/guns/guns_by_map/desert/guns_obj.dmi",
```
Comment (DrSmugleaf) [inline Resources/Textures/_RMC14/Objects/Weapons/Guns/Shotguns/m890.rsi/meta.json:4]: The in-hands are not from this dmi alone
```suggestion
  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/7cb618c69b75873f3ce893022fe08d1233b3152d/icons/obj/items/weapons/guns/guns_by_map/desert/guns_obj.dmi, https://github.com/cmss13-devs/cmss13/blob/106c92cdf232ebc12c9d7a2feb23956c6755496f/icons/mob/humans/onmob/items_lefthand_1.dmi, https://github.com/cmss13-devs/cmss13/blob/4047c473d4d2f3e58e68efdec5aa95d0676bf4d1/icons/mob/humans/onmob/items_righthand_1.dmi",
```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by Rainbeon** [comment]
This has blown up the YAML linter and RSI validator

**Pull request merged**